Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizationDemo.cs

```diff
@@ -105,7 +102,7 @@ private void AddMarkers()
             float widthRect;
             float heightRect;
 
-            if (!BoundaryManager.TryGetRectangularBoundsParams(out centerRect, out angleRect, out widthRect, out heightRect))
+            if (!MixedRealityManager.BoundarySystem.TryGetRectangularBoundsParams(out centerRect, out angleRect, out widthRect, out heightRect))
```

Comment:
Not checking for no boundary system
```
MixedRealityManager.BoundarySystem?.
```

Reply:
It didn't before

Reply:
I'm going to ignore the fact you just said that :P

Reply:
I didn't want to make any logical changes, just refactor.

Reply:
I really should have just updated all the root references to the different systems, but I wanted people to use the ideal way to access the core systems via the Mixed Reality Manager.

Reply:
We check for null before we call `AddMarkers`